pkg/reconciler/pipelinerun/resources/pipelinerunstate.go

```diff
@@ -71,6 +72,23 @@ func (state PipelineRunState) IsBeforeFirstTaskRun() bool {
 	return true
 }
 
+// AdjustStartTime adjusts potential drift in the PipelineRun's start time.
```

Comment:
In addition to stating why we do this, can you add a sentence describing how we adjust, to make this easier to read?

Reply:
tweaked wording, PTAL 🙏